### DISCUSSION TITLE
How many 🍋 are in this grid?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#507] [Galaxy Trivia] [answered]
```
N O M E L L E M O N L E M O N
N O M E L N O M E L E L E M O
L E M O N N O M E L L E M O M
L O E L O L L M M L E M O N E
N N O M E L O E L E M O N E L
O L E M L N L L M M L N L L E
M L O E L N L E M O N E E E M
E N M E O E E M N N N M M M O
L O M M N N M N L O O L L O N
N O E L O M O O N E M E L N N
N L N E M E N M N O M E L O N
O N O M E L O E E O M O L M O
M L M O L E M L N L E E N E M
E E E N O M E L E M O N L L E
L M L O M E L E M O N O M E L
```

**Comment by peterbaccaro** [comment]
A lot of lemons. :laughing:

**Reply by peterbaccaro** [reply]
![lemons](https://user-images.githubusercontent.com/74666079/230712078-caa007f4-f0f4-4c90-890a-a92767612847.png)

I think 42 lemons.

**Reply by ParanoidUser** [reply]
Great harvest! But some tricky 🍋s have eluded you. Could you spot them? 🔭

*Hint: all are diagonal*

**Reply by peterbaccaro** [reply]
![lemons](https://user-images.githubusercontent.com/74666079/230739646-0e2cdce2-8b71-493a-97d2-f3b931365e50.png)

I found two more, so that would make it 44 lemons.

**Reply by ParanoidUser** [reply]
**Bad news:** there are more! 🤫
**Good news:** they all on the left half. 🫣

**Reply by peterbaccaro** [reply, marked as answer]
![lemons](https://user-images.githubusercontent.com/74666079/230761786-c095cd94-e6ae-449a-9e02-9ae51b48a015.png)

I found three more so hopefully I have found them all now.  That would make it 47 lemons.

**Reply by ParanoidUser** [reply]
You nailed it! I knew you could do it, hope it wasn't too much of a squeeze. 😝

**Comment by thuongtruong1009** [comment]
There are 13 🍋 in the grid.
